step-certificates/examples/certificate_authority_single_instance/values.yml.tpl

```
@@ -0,0 +1,130 @@
+kind: StatefulSet
+replicaCount: 1
+autocert:
+  enabled: false
+bootstrap:
+  enabled: false
+ca:
+  db:
+      enabled: true
+      persistent: false
+inject:
+  enabled: true
+  config:
+    files:
+      ca.json:
+        root: /home/step/certs/root_ca.crt
+        federateRoots: []
+        crt: /home/step/certs/intermediate_ca.crt
+        key: /home/step/secrets/intermediate_ca_key
+        address: 0.0.0.0:9000
+        dnsNames: ${CA_DNS_NAMES}
+        logger:
+          format: json
+        db:
+          type: badger
```

We've updated this by default to `badgerv2`

```suggestion
          type: badgerv2
```